static/scripts/teamMembers.js

```diff
@@ -355,8 +354,18 @@ $(document).ready(() => {
 		e.stopPropagation();
 		e.preventDefault();
 		const $editMemberModal = $('.edit-member-modal');
-		const userId = $(this).parent().parent().find('[data-payload]')
+		const payload = $(this).parent().parent().find('[data-payload]')
 			.data('payload');
+		if (process.env.FEATURE_STUDENTS_CANT_BE_TEAM_ADMINISTRATOR) {
```

Comment:
does this work? The env should be not avalible here, because this run in the browser.